webview/src/plots/components/ribbon/RibbonBlockTooltip.tsx

```diff
@@ -8,20 +8,26 @@ import { Icon } from '../../../shared/components/Icon'
 import Tooltip from '../../../shared/components/tooltip/Tooltip'
 import { CopyButton } from '../../../shared/components/copyButton/CopyButton'
 import { GitCommit } from '../../../shared/components/icons'
+import { ErrorTooltipContent } from '../../../shared/components/tooltip/ErrorTooltip'
 
 export const RibbonBlockTooltip: React.FC<{
   revision: Revision
   children: ReactElement
 }> = ({ revision, children }) => {
-  const { firstThreeColumns, commit } = revision
+  const { firstThreeColumns, commit, errors } = revision
 
   const tooltipContent = (
     <div>
+      {errors && (
+        <div className={styles.addBottomBorder}>
+          <ErrorTooltipContent error={errors.join('\n')} />
+        </div>
+      )}
       <table className={styles.columnsTable}>
         <tbody>
           {firstThreeColumns.map(({ path, value, type }) => (
             <tr key={path}>
-              <td className={cn(styles[`${type}Key`])}>
+              <td className={cx(styles[`${type}Key`])}>
                 {truncate(path, 45, 'left')}
```

Comment:
Out of scope and I must have been the one who did this. But certainly, truncating in Javascript is a last resort. That should be done in CSS instead. If it is absolutely necessary there should have been a comment explaining the reasoning of using `vega-util` `truncate` function here. I would even add that if truncating in Javascript is the way to go, using a library for plotting is probably not the way to go either. I'll take a not and investigate.